Two-body-scf-guess.py

```diff
@@ -0,0 +1,71 @@
+from wrap_pyscf import *
+import numpy as np
+
+
+mol = [("O", 0.216486, -1.989005, 0.035297),
+       ("H", -0.603911, -1.505274, -0.033552),
+       ("H", 0.138672, -2.535875, 0.799566),
+       ("H", 1.505276, -0.603920, 0.033505),
+       ("O", 1.989004, 0.216483, -0.035296),
+       ("H", 2.535922, 0.138690, -0.799534),
+       ("O", -1.989001, -0.216487, -0.035297),
+       ("H", -2.535947, -0.138688, -0.799513),
+       ("H", -1.505273, 0.603920, 0.033493),
+       ("O", -0.216485, 1.989007, 0.035294),
+       ("H", -0.138694, 2.535879, 0.799565),
+       ("H", 0.603919, 1.505277, -0.033513)] 
+
+
+def two_body_guess(monomers, dimers, bs):
+    y = len(monomers)
+    for dimers_index in dimers:
+        dimers_index = dimers.index(dimers_index)
```

Comment:
I highly recommend not overwriting your loop variable like this. If you want the element and the index use `enumerate` like:

```
for i, dimer in enumerate(dimers):
    print("Dimer {} = {}".format(i, dimer))
```